src/analysis/normed_space/extend.lean

```diff
@@ -0,0 +1,161 @@
+/-
+Copyright (c) 2020 Ruben Van de Velde All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Ruben Van de Velde
+-/
+
+import analysis.complex.basic
+import analysis.normed_space.operator_norm
+
+/-!
+# Extending a continuous ℝ-linear map to a continuous ℂ-linear map
+
+In this file we provide a way to extend a continuous ℝ-linear map to a continuous ℂ-linear map in
+a way that bounds the norm by the norm of the original map.
+
+We motivate the form of the extension as follows. Note that `fc : F →ₗ[ℂ] ℂ` is defined fully by
```

Comment:
```suggestion
We motivate the form of the extension as follows. Note that `fc : F →ₗ[ℂ] ℂ` is determined fully by
```
(when we're doing type theory, "defined" means something a bit more specific than in usual mathematics!)